Userland/Libraries/LibJS/Tests/builtins/ArrayBuffer/ArrayBuffer.prototype.resize.js

```diff
@@ -0,0 +1,33 @@
+test("resize", () => {
+    expect(
+        (() => {
+            let a = new ArrayBuffer(0, { maxByteLength: 10 });
+            a.resize(10);
+            return a.byteLength;
+        })()
+    ).toEqual(10);
```

Comment:
Instead of testing with IIFEs we usually use `describe("normal behavior")` and `test("test name" , () => {})`
Like for example:
```js
test("sets byteLength on resize", () => {
    const a = new ArrayBuffer(0, { maxByteLength: 10 });
    a.resize(10);
    expect(a.byteLength).toEqual(10);
});
```

This also gives a little information in the name of the string what/why it is testing.